lib/chai/assertion.js

```diff
@@ -42,139 +39,135 @@ import * as util from './utils/index.js';
  *
  * - `eql`: This flag contains the deepEqual function to be used by the assertion.
  *
- * @param {unknown} obj target of the assertion
- * @param {string} msg (optional) custom error message
- * @param {Function} ssfi (optional) starting point for removing stack frames
- * @param {boolean} lockSsfi (optional) whether or not the ssfi flag is locked
+ * @param {unknown} ?obj target of the assertion
```

Comment:
we should use typescript's jsdoc format

```ts
/**
 * @param {unknown=} obj some optional object
 */
```

Reply:
Yeah, I realised that this is wrong and fix it in #1679